### PR TITLE
docs: Changelog for 1.0.0

### PR DESCRIPTION
**Pull request by frondeus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,53 +1,63 @@
...
+- Partial Update Engine (PUE) - Notifies OB when materialized view needs to be processed
 
-### Changes to existing components:
-- Schema Registry now uses Postgres Backend
-- CDL can now work in ordered (messages) mode
-- CDL now uses Open Telemetry
-- Data Router supports Message Batching
+### Changes to existing components
+- Data Router and Query Router supports static routing allowing Schema-Registry-absent deployment
+- Query Service supports returning stream of data without allocating hughe amounts of memory
```
Comment (Kononnable) [inline CHANGELOG.md:23]:
```suggestion
- Query Service supports returning stream of data without allocating huge amounts of memory
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
-- Materializer - general (M-G)
-- CDL Web Admin (GUI)
-- Edge Registry - (ER)
-- Partial Update Engine - (PUE)
+### New components
+- CDL Web Admin (GUI) - For management purposes only
+- Edge Registry (ER) - Manages relationships between schemas (schema to schema) and objects (object to object)
+    - ER is available in graphQL API
+    - ER pushes notifications
+- Materializer - general (M-G) - Writes materialized data to the repository
```
Comment (luke-biel) [inline CHANGELOG.md:12]: Word `repository` may be misleading

Comment (luke-biel) [reply CHANGELOG.md:12]:
```suggestion
- Materializer - general (M-G) - Writes materialized data to the materialization-database
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,53 +1,63 @@
...
 
-### General Changes:
-- Object Builder is available in Graphql API
-- Object Builder supports empty filters
-- Object Builder can use filters in queries
-- Edge Registry is available in Graphql API
-- Edge Registry notifications support added
-- All services can be queried with --help to get full list of options with explanations
+### General changes
+- E2E tests (E2E)
```
Comment (Kononnable) [inline CHANGELOG.md:28]: Completely internal, I don't think we should put it in a changelog.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,53 +1,63 @@
...
 
-### General Fixes
-- Consumer-based services will log error and continue on invalid message. no longer resulting in crash loop
-- Query multiple returns error for timeseries
-- build script fix
-- Metrics port defaulted to invalid ones
+### Geneal fixes
+- Query-multiple RPC method returns an error for Timeseries schema type
+- Metrics port defaulted to invalid one
+- Build script fixes
```
Comment (Kononnable) [inline CHANGELOG.md:44]: Internal

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,53 +1,63 @@
...
+- CDL Ingestion API versioning
+- CDL publishing deployment configurations
+- Partial Materialization
 - Edge Registry
-- Initial draft of the MessagePack
-- Alternative communication method to Kafka and RabbitMQ
+- Alternative communication method to Kafka and RabbitMQ (gRPC)
+
+### Known issues
+- Relationships in materialization may generate wrong set of data
```
Comment (Kononnable) [inline CHANGELOG.md:55]: Should we put it here? Officially relationships in materialization aren't supported yet.

**Review by Kononnable — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (frondeus) [reply CHANGELOG.md:55]: But still, the code is there, it's not under any feature flag, and someone may use it. So it's better to be safe than sorry by having "We know about it, don't bother us".

**Review by esavier — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (frondeus) [reply CHANGELOG.md:54]: I may rephrase it "Relationships in materialization are still WIP and may generate wrong set of data"

**Review by Kononnable — APPROVED** [review]

**Pull request merged**